client/components/LayoutEditor/LayoutEditorPubs/LimitPubs.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { RadioGroup, Radio } from '@blueprintjs/core';
+
+import { SliderInput } from 'components';
+import { maxPubsPerBlock } from 'utils/layout';
+
+type Props = {
+	limit?: number;
+	onChangeLimit: (nextLimit: undefined | number) => unknown;
+};
+
+const defaultCustomLimit = 10;
+
+const LimitPubs = (props: Props) => {
+	const { limit, onChangeLimit } = props;
+	const [intermediateLimit, setIntermediateLimit] = useState(limit || defaultCustomLimit);
+
+	const handleRadioSelect = (value: string) => {
+		if (value === 'all') {
+			onChangeLimit(undefined);
+		} else {
+			onChangeLimit(defaultCustomLimit);
+			setIntermediateLimit(defaultCustomLimit);
+		}
+	};
+
+	const handleChange = (value: number, committed: boolean) => {
+		setIntermediateLimit(value);
+		if (committed) {
+			onChangeLimit(value);
+		}
+	};
+
+	return (
+		<div className="layout-editor-pubs_limit-pubs-component">
+			<RadioGroup
+				onChange={(evt) => handleRadioSelect(evt.currentTarget.value)}
+				selectedValue={limit ? 'limit' : 'all'}
+			>
+				<Radio value="all" label={`Show up to ${maxPubsPerBlock} Pubs`} />
```

Comment:
Makes sense to keep the `all` vs `limit` separation since that's how we model it at the moment. I think a language update could make it feel less redundant though: "Show max number of pubs (100)".